/var/lib/jenkins/home/jobs/MITK-PR/workspace/Modules/Utilities/include/TranslationUtilities.h:12:22: error: ‘string’ is not a member of ‘std’
   static std::vector<std::string> getTranslatedTransferFunctionPresets();
                      ^~~

--- a/Modules/Utilities/include/TranslationUtilities.h
+++ b/Modules/Utilities/include/TranslationUtilities.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <vector>
 
 #include <MitkUtilitiesExports.h>

